astroquery/conftest.py

```diff
@@ -47,3 +47,13 @@ def pytest_configure(config):
 packagename = os.path.basename(os.path.dirname(__file__))
 TESTED_VERSIONS[packagename] = version
 TESTED_VERSIONS['astropy_helpers'] = astropy_helpers_version
+
+
+def pytest_addoption(parser):
+    parser.addoption(
```

Comment:
again, this is nit pick, we don't use 'black', formatting so no need to put every arg in a new line 😄